pypit/_version.py

```diff
@@ -0,0 +1,2 @@
+__version__ = '0.6.dev0'
```

Comment:
I know that you've used _version.py below in pyputils, but should the name of this file be `__version__.py`, in the same way that we have a `__init__.py` ?